common/src/test/scala/org/neo4j/spark/util/Neo4jOptionsTest.scala

```diff
@@ -204,4 +204,20 @@ class Neo4jOptionsTest {
     assertEquals(URI.create("neo4j://localhost"), baseUrl)
     assertEquals(Set(ServerAddress.of("foo.bar", 7687), ServerAddress.of("foo.bar.baz", 7783)), resolvers)
   }
+
+  @Test
+  def testGdsProperties(): Unit = {
```

Comment:
it may be worth adding an integration test with invalid options (partitions > 1, write/mutate mode, etc)
WDYT?

Reply:
done, please check `GraphDataScienceIT.shouldFailWithUnsupportedOptions`